site/content/docs/main/config/upstream-tls.md

```diff
@@ -5,7 +5,7 @@ Applying the `projectcontour.io/upstream-protocol.tls` annotation to a Service o
 The same configuration can be specified by setting the protocol name in the `spec.routes.services[].protocol` field on the HTTPProxy object.
 If both the annotation and the protocol field are specified, the protocol field takes precedence.
 By default, the upstream TLS server certificate will not be validated, but validation can be requested by setting the `spec.routes.services[].validation` field.
-This field has mandatory `caSecret` and `subjectName` fields, which specify the trusted root certificates with which to validate the server certificate and the expected server name.
+This field has mandatory `caSecret` and `subjectName` fields, which specify the trusted root certificates with which to validate the server certificate and the expected server name. The `caSecret` can be a namespaced name of the form `<namespace>/<secret-name>`. If the CA secret's namespace is not the same namespace as the `HTTPProxy` resource, [TLS Certificate Delegation][4] must be used to allow the owner of the CA certificate secret to delegate, for the purposes of referencing the CA certificate in a different namespace, permission to Contour to read the Secret object from another namespace.
```

Comment:
tiny nit: Could you put each sentence on its own line? Makes it so future changes are easier to read in a diff

Reply:
@sunjayBhatia I thought I made sense to describe the `caSecret` requirement in the same paragraph. If this doesn't seem correct, I'd be happy to change it.

Reply:
the rendered text will all be in the same paragraph with the sentences on their own line in the file with no extra line between them since it’s markdown (similar to the previous text this change appends to)

Reply:
I get what you mean now, will do!